src/main/scala/view/scalaFX/components/WolfView.scala

```diff
@@ -0,0 +1,79 @@
+package view.scalaFX.components
+
+import scalafx.animation.{AnimationTimer, KeyFrame}
+import scalafx.scene.image.{Image, ImageView}
+import view.scalaFX.ScalaFxViewConstants.{PANEL_SKY_ZONE, PREFERRED_BUNNY_PANEL_HEIGHT, PREFERRED_BUNNY_PANEL_WIDTH}
+import view.scalaFX.utilities.Direction
+import view.scalaFX.utilities.Direction._
+
+import scala.language.postfixOps
+import scala.util.Random
+
+/** Bunny wrapper in order to manage its movement inside of the GUI */
+trait WolfView {
+  /** Reference to the model disturbing factor entity */
+//  val wolf: DisturbingFactor
+
+  /** The image of the disturbing factor displayed on the GUI */
+  val imageView: ImageView
+
+  /** Type annotation for a Seq of KeyFrames */
+  type AnimationFrames = Seq[KeyFrame]
+
+  /** Starts the disturbing factor animation */
+  def play(): Unit
+}
+
+object WolfView {
+
+  def apply(): WolfView = {
+    val newX = Random.nextInt(PREFERRED_BUNNY_PANEL_WIDTH)
```

Comment:
Per questa costante potremmo cambiare nome in solo "PREFERRED_PANEL_WIDTH" o "PREFERRED_SIMULATION_PANEL_WIDTH" ( e lo stesso per l'altezza)